src/util/chunk_fs.js

```diff
@@ -62,6 +62,8 @@ class ChunkFS extends stream.Transform {
         this._flush_buffers(callback);
     }
 
+    // callback will be passed only at the end of the stream
+    // while this function can be called sooner without callback
```

Comment:
just a minor comment clarification
```suggestion
    // callback will be passed only at the end of the stream by _flush()
    // while this function is called without callback during _transform() and returns a promise.
```